### DISCUSSION TITLE
How to change pagination color when using Svelte and Tailwind CSS?

### DISCUSSION THREAD

**Discussion by jaads** [nolimits4web/swiper#5969] [Q&A]
Hey I want to change the default blue pagination color to a different one.
I'm using Svelte / Sveltekit and Tailwind CSS.
After adding the attributes to the pagination parameter object, the pagination just disappears.

Any ideas?

```
<Swiper
    modules={[Navigation, Pagination, A11y]}
    spaceBetween={10}
    navigation
    pagination={{
      clickable: true,
      bulletClass: "w-2.5 h-2.5 block rounded-full cursor-pointer",
      bulletActiveClass: "!bg-green-500",
      bulletInActiveClass: "!bg-yellow-500",
      el: "#pagination",
    }}
  >
    {#each imageNames as image}
      <SwiperSlide>
        <img src={`/folder/${image}.jpg`} />
      </SwiperSlide>
    {/each}
  </Swiper>
```

**Comment by dninomiya** [comment]
@jaads 
`bulletInActiveClass` is a key that does not exist.
That's the key I'm proposing to add, which doesn't currently exist. #5869 

Try removing "bulletInActiveClass".
